OmsiHook/OmsiStructs.cs

```diff
@@ -1832,4 +1878,22 @@ public struct RTL_CRITICAL_SECTION
         public uint SpinCount;
     }
 
+    public struct OmsiWString
```

Comment:
If you're gonna implement `OmsiWString` please at least also do `OmsiString` as well.